packages/gatsby/package.json

```diff
@@ -110,6 +110,7 @@
     "got": "^11.8.5",
     "graphql": "^16.6.0",
     "graphql-compose": "^9.0.9",
+    "graphql-tag": "^2.12.6",
```

Comment:
This is already being installed transitively (codegen deps eventually install it), so it's not adding new dependency